doc/lua_api.txt

```diff
@@ -1532,7 +1532,8 @@ Another example: Make red wool from white wool and red dye:
 Special groups
 --------------
 
-* `immortal`: Disables the group damage system for an entity
+* `immortal`: Disables the group damage system for an entity.
+  Disables all damage for a player.
```

Comment:
`Skips all damage and breath handling for a player. This group will also hide the integrated HUD status bars.`